VRDR/DeathRecord.cs

```diff
@@ -5819,6 +5819,46 @@ public string DeathTime
             }
         }
 
+        /// <summary>DateOfDeathDeterminationMethod.</summary>
+        /// <value>method. A Dictionary representing a code, containing the following key/value pairs:
+        /// <para>"code" - the code</para>
+        /// <para>"system" - the code system this code belongs to</para>
+        /// <para>"display" - a human readable meaning of the code</para>
+        /// </value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>Dictionary&lt;string, string&gt; code = new Dictionary&lt;string, string&gt;();</para>
+        /// <para>code.Add("code", "exact");</para>
+        /// <para>code.Add("system", CodeSystems.DateOfDeathDeterminationMethods);</para>
+        /// <para>code.Add("display", "exact");</para>
+        /// <para>ExampleDeathRecord.DateOfDeathDeterminationMethod = code;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Date of Death Determination Method: {ExampleDeathRecord.DateOfDeathDeterminationMethod['display']}");</para>
+        /// </example>
+
+       [Property("DateOfDeathDeterminationMethod", Property.Types.Dictionary, "Death Investigation", "Date of Death Determination Method.", true, IGURL.DeathDate, true, 25)]
```

Comment:
Should this be `IGURL.DeathDate` or should it use the `CodeSystems` IG URL that was added?

Reply:
The annotation references the profile that this data element is part of....right @pkmitre ?

Reply:
IGURL.DeathDate is correct